amundsen_application/static/js/features/ColumnList/ColumnLineage/index.tsx

```diff
@@ -0,0 +1,90 @@
+// Copyright Contributors to the Amundsen project.
+// SPDX-License-Identifier: Apache-2.0
+
+import { connect } from 'react-redux';
+
+import { GlobalState } from 'ducks/rootReducer';
+import * as React from 'react';
+import { emptyLineage } from 'ducks/tableMetadata/reducer';
+
+import { getColumnLineageLink } from 'config/config-utils';
+import './styles.scss';
+import { Lineage, TableMetadata } from 'interfaces/TableMetadata';
```

Comment:
Nit: I would order it differntly:

```suggestion
import * as React from 'react';
import { connect } from 'react-redux';

import { GlobalState } from 'ducks/rootReducer';
import { emptyLineage } from 'ducks/tableMetadata/reducer';
import { getColumnLineageLink } from 'config/config-utils';

import { Lineage, TableMetadata } from 'interfaces/TableMetadata';

import './styles.scss';

```

Reply:
One of the artifacts of using "alt+enter" to import a dep is that it seems to place it somewhat randomly. Will clean up.